Dockerfile

```diff
@@ -9,7 +9,7 @@ ADD ./zebedee-cms/target/classes /usr/src/target/classes
 EXPOSE 9200
 
 # Update the entry point script
-ENTRYPOINT java -Xmx2048m \
+ENTRYPOINT java -Xmx1204m -Xdebug -Xrunjdwp:transport=dt_socket,address=8002,server=y,suspend=n \
```

Comment:
Few comments - 
1. why the reduction in heap space? 
2. seems to be some pointers that -Xrunjdwp is for pre Java 5. `-agentlib:jdwp` is the more modern way.
3. you may want to make the address configurable with a default for future conflicts
4. please make sure this is documented in the readme

Reply:
Made some changes and seems to be working now also with the port configurable.